examples/lib/defaultHtmlStepUi.js

```diff
@@ -182,14 +184,14 @@ function DefaultHtmlStepUi(_sequencer, options) {
       }
     }
     else {
-      $('#load-image').append(step.ui);
+      $('#load-image').append(step.ui); // Default UI without extra tools for the first step(load image).
     }
-    $step('.toggle').on('click', () => {
+    $step('.toggle').on('click', () => { // Step container dropdown
       $step('.toggleIcon').toggleClass('rotated');
       $stepAll('.cal').collapse('toggle');
     });
     
-    $(step.imgElement).on('mousemove', _.debounce(() => imageHover(step), 150));
+    $(step.imgElement).on('mousemove', _.debounce(() => imageHover(step), 150)); // Shows the pixel coordinates on hover.
```

Comment:
If you are writing X shows the pixel coordinates on hover then you have to add .
So, remove .

Reply:
```suggestion
    $(step.imgElement).on('mousemove', _.debounce(() => imageHover(step), 150)); // Shows the pixel coordinates on hover
```